src/app/layout/layout.style.ts

```diff
@@ -1,12 +1,15 @@
 import styled from 'styled-components';
 
-const Container = styled.div`
+export const Container = styled.div`
+  min-height: calc(100% - 27.3rem);
+`;
+
+export const Wrapper = styled.div`
```

Comment:
```suggestion
export const GridWrapper = styled.div`
```
처럼 Grid 시스템을 제어하는 Wrapper 라는 표현을 해주는 건 어떨까요?

Reply:
그게 더 명확하겠네요 수정했습니다~ https://github.com/GrebbMe/grabbme-fe/pull/62/commits/24cec2b0272c5b746e115754c3988643c9d3af49